### PR TITLE
Bug - WSOD on activity log page #6147

### PR DESCRIPTION
**Pull request by sergiogcx**
Closes https://github.com/cityofaustin/atd-data-tech/issues/6147

Sorry for the extra diff noise, it looks like prettier went to town on this one.

Live Preview:
https://deploy-preview-314--atd-moped-main.netlify.app/moped/projects/176?tab=activity_log

### CODE REVIEW

**Review by chiaberry — APPROVED** [review]

**Review by amenity — APPROVED** [review]
LSTM 👌

File: moped-editor/src/views/projects/projectView/ProjectActivityLog.js [hunk 1]
```diff
@@ -97,15 +98,36 @@ const ProjectActivityLog = () => {
...
+
+  /**
+   * Retrieve the user's full name or return an "N/A"
+   * @param {object} moped_user - The user object as provided by hasura
+   * @return {string}
+   */
+  const getUserFullName = moped_user => {
+    const firstName = moped_user?.first_name ?? "";
+    const lastName = moped_user?.last_name ?? "";
+    if (firstName.length === 0 && firstName.length === 0)
```
Comment (mateoclarke) [inline moped-editor/src/views/projects/projectView/ProjectActivityLog.js:112]: 👍

File: moped-editor/src/views/projects/projectView/ProjectActivityLog.js [hunk 2]
```diff
@@ -97,15 +98,36 @@ const ProjectActivityLog = () => {
...
    * @return {string}
    */
-  const getInitials = name =>
-    name
+  const getInitials = moped_user => {
+    // Get any names if available
+    const name = getUserFullName(moped_user).trim();
+
+    // If no names are available, return null to force the generic humanoid avatar
+    if (name.length === 0 || name === unknownUserNameValue) return null;
```
Comment (mateoclarke) [inline moped-editor/src/views/projects/projectView/ProjectActivityLog.js:127]: 👍

**Review by mateoclarke — APPROVED** [review]
hi_my_name_is

🚢

**Pull request merged**